lab-davide/gulpfile.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+
+
+
+gulp.task('eslint', function() {
+  return gulp.src('server.js')
+    .pipe(eslint(
```

Comment:
No more eslint configs in your gulp file! 

create a file called .eslintrc in your root project directory and put this in there.